src/panner.js

```diff
@@ -74,46 +74,38 @@ if (typeof ac.createStereoPanner !== 'undefined') {
         delete this.stereoPanner;
       }
     }
-
-    //not implemented because stereopanner
-    //node does not require this and will automatically
-    //convert single channel or multichannel to stereo.
-    //tested with single and stereo, not with (>2) multichannel
-    inputChannels() {}
   }
 
   panner = Panner;
 } else {
   // if there is no createStereoPanner object
   // such as in safari 7.1.7 at the time of writing this
   // use this method to create the effect
-  class Panner {
-    constructor(input, output, numInputChannels) {
-      this.input = ac.createGain();
-      input.connect(this.input);
+  class Panner extends Effect {
+    constructor() {
+      super();
+
+      // 'explicit' channelCountMode will convert any number of channels to stereo
+      this.input.channelCountMode = 'explicit';
```

Comment:
Do we have to set `input.channelCount` for this? Just based on the docs here: https://developer.mozilla.org/en-US/docs/Web/API/AudioNode/channelCountMode